landlab/components/landslides/__init__.py

```diff
@@ -1,5 +1,5 @@
 
-from .landslide import LandslideProbability
```

Comment:
There should not be. The name of the file 'landslides_probability.py' has been changed from 'landslides.py'. But, there is an .__init__ file in that folder that should take care of this. This should be a low level (internal/away from the user) interface. Please correct me if I am wrong.